src/state/metadata/logics.ts

```diff
@@ -272,11 +281,17 @@ const loadFileViaUrl = createLogic({
                 done();
             })
             .catch((error) => {
+                let errorDetails = `"${url}" failed to load.`;
+                // If there was a CORS error, error.message does not contain a status code
+                if (error.message === "Failed to fetch") {
+                    errorDetails +=
+                        " Try uploading your trajectory file from a Dropbox or Amazon S3 link instead.";
```

Comment:
I think you can pass html to this error, if I'm right about that, I would put this on a new line, because as it it, I might miss it thinking it just a bunch of error text.

Reply:
Agreed, I just put in 2 line breaks to make it stand out extra. Better?
![image](https://user-images.githubusercontent.com/12690133/121603946-81b60d00-c9fe-11eb-89b2-80d97054dae6.png)